docs/CPP_STYLE.md

```diff
@@ -191,15 +193,15 @@ v = w * (x + z);
 
 ## Tips and Hints
 
-- If you see outdated code which can be improved - DO IT NOW (but in the separate pull request)! Make this awesome world even more better! 
+- If you see outdated code which can be improved - DO IT NOW (but in a separate pull request)!
 - Your code should work at least on [mac|win|linux|android][x86|x86_64], [ios|android][armv7|arm64] architectures
 - Your code should compile well with gcc 4.8+ and clang 3.5+
```

Comment:
gcc 4.8 doesn't support c++14 asaik (at least according to this https://gcc.gnu.org/projects/cxx-status.html)
So it looks like the minimal version which supports all c++14 features is 5.x.

Reply:
gcc 6.0, clang ?